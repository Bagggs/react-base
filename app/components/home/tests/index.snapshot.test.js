import React from 'react';
import renderer from 'react-test-renderer';
import withI18n from 'i18n-test';
import { Home } from '../';

describe('Home', () => {
  it('renders correctly', () => {
    const homeComponent = renderer.create(withI18n(<Home/>)).toJSON();

    expect(homeComponent).toMatchSnapshot();
  });
});
