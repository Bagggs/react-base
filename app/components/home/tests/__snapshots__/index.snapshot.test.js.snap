// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="jumbotron"
  >
    <h1>
      React-base
    </h1>
    <p>
      <span>
        <span
          dangerouslySetInnerHTML={
            Object {
              "__html": "Kick-start your new web application based on React and Redux technologies.<br/>It also includes Webpack, React hot loader, PostCSS, JSON-server tools for even more rapid development.",
            }
          }
        />
      </span>
    </p>
  </div>
</div>
`;
